### PR TITLE
This commit should allow us use the engage API for Helpdesk whatsapp …

### PR DESCRIPTION
**Pull request by DevChima**

### CODE REVIEW

File: casepropods/family_connect_registration/plugin.py [hunk 1]
```diff
@@ -100,20 +98,21 @@ def get_identity_registration_data(self, identity, registrations):
             })
         return items
 
-    def has_whatsapp_account(self, number):
+    def has_whatsapp_account(self, number, wait_for_response):
```
Comment (erikh360) [inline casepropods/family_connect_registration/plugin.py]: Doesn't look like `wait_for_response` is used anywhere in the function, can probably remove it.

File: .travis.yml [hunk 2]
```diff
@@ -25,6 +23,7 @@ before_install:
   - psql -U postgres -c "CREATE USER casepro WITH PASSWORD 'nyaruka';"
   - psql -U postgres -c "ALTER ROLE casepro WITH SUPERUSER;"
   - psql -U casepro postgres -c "CREATE DATABASE casepro;"
+  - psql -U casepro postgres -c "SELECT version();"
```
Comment (erikh360) [inline .travis.yml]: We can remove this debug line.

**Review by erikh360 — COMMENTED** [review]
Awesome!, just two small comments

File: casepropods/family_connect_registration/plugin.py (hunk 1 above)
Comment (DevChima) [reply casepropods/family_connect_registration/plugin.py]: Thanks @erikh360 
I thought the API works with two parameters (contacts (mandatory) and blocking (optional)). 
Aren't we supposed to pass in "wait" or "no_wait" here so the application knows how the process the request?

"blocking": wait_for_response

 It is "no_wait" by default so if you feel it's still fine to take out I can do that.

Comment (erikh360) [reply casepropods/family_connect_registration/plugin.py]: Oh now I see where it is used. I still don't think there is a need for the variable, just put the "wait" string directly in your payload.

**Comment by DevChima** [comment]
@erikh360 Ready for another review.

File: casepropods/family_connect_registration/tests.py [hunk 3]
```diff
@@ -39,9 +39,9 @@ def setUp(self):
                 'identity_store_token': 'identity-store-token',
                 'stage_based_messaging_url': 'http://sbm/api/v1/',
                 'stage_based_messaging_token': 'sbm-token',
-                'wassup_url': 'http://wassup/api/v1',
-                'wassup_token': 'wassup-token',
-                'wassup_number': '+27821234567',
+                'engage_url': 'https://engage.example.org/v1/contacts',
+                'engage_api_token': 'rapidprotoken',
+                'engage_number': '+27821234567',
```
Comment (erikh360) [inline casepropods/family_connect_registration/tests.py]: I don't think engage_number is needed.

**Comment by DevChima** [comment]
@erikh360 Ready for another one. Please let me know if you'd like me to change something else. Thanks

**Review by erikh360 — APPROVED** [review]
👍

**Pull request merged**